.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,9 @@
+name: Pull Request CI
+
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
We used to run CI on all pull requests. For now I think it makes sense to just run it on main, but I could be convinced otherwise.